src/KM_Utils.pas

```diff
@@ -674,6 +676,23 @@ function GetNoColorMarkupText(aText: UnicodeString): UnicodeString;
 end;
 
 
+//Replace continious spaces with single space
+function DeleteDoubleSpaces(aString: string): string;
+var I: Integer;
+begin
+  Result := '';
+  for I := 1 to Length(aString) do
+  begin
+    if aString[I] = ' ' then
+    begin
+      if not (aString[I-1] = ' ') then

```

Comment:
On the first iteration this could check aString[0] which is out of bounds and will crash. You should start the iteration from 2 instead.